{"uuid": "69b4c18a-c30e-4850-a9d6-7181cffa8d2f", "children": ["c48e76f6-f471-47c7-817d-7253316196dc"], "befores": [{"name": "login_success", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\yiii\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\yiii\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\yiii\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"E:\\Jenkins_Slave_Home\\workspace\\ht_web_pytest\\TestCases\\conftest.py\", line 61, in login_success\n    LoginPage(driver).login(LD.success_data[\"用户名\"], LD.success_data[\"密码\"])\n  File \"E:\\Jenkins_Slave_Home\\workspace\\ht_web_pytest\\PageObjects\\login_p.py\", line 13, in login\n    self.wait_eleVisible(loc.name_text)\n  File \"E:\\Jenkins_Slave_Home\\workspace\\ht_web_pytest\\Common\\basepage.py\", line 42, in wait_eleVisible\n    self.save_screenshot(doc)\n  File \"E:\\Jenkins_Slave_Home\\workspace\\ht_web_pytest\\Common\\basepage.py\", line 33, in wait_eleVisible\n    WebDriverWait(self.driver, wait_times, poll_frequency).until(EC.visibility_of_element_located((locator[1])))\n  File \"c:\\users\\yiii\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1587375146375, "stop": 1587375188753}], "start": 1587375146375, "stop": 1587375188807}